### PR TITLE
Fix typo and adjust import order

### PR DESCRIPTION
**Pull request by hezhizhen**
This PR is a 
 - [ ] Feature Implementation
 - [x] Bug Fix
 - [ ] Documentation

**What this PR does / why we need it**:

Fix some typos and adjust import order based on goland suggestions.

**Which issue(s) this PR fixes** 

None.

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

**Review by duanhongyi — APPROVED** [review]
LGTM

File: cmd/svcat/command/command.go [hunk 1]
```diff
@@ -17,15 +17,14 @@ limitations under the License.
...
+	"fmt"
 	"strings"
 	"unicode"
 
-	"fmt"
-
 	"github.com/spf13/cobra"
 )
 
-// Command represents an svcat command.
```
Comment (jhvhs) [inline cmd/svcat/command/command.go]: an SVCAT is actually correct. See https://www.languageediting.com/how-to-choose-the-indefinite-article-a-or-an-before-acronyms/

File: cmd/svcat/instance/provision_cmd_test.go [hunk 2]
```diff
@@ -23,16 +23,16 @@ import (
...
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/kubernetes-sigs/service-catalog/pkg/svcat"
 	servicecatalog "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
+	servicecatalogfakes "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/instance/provision_cmd_test.go]: If we are introducing an alias for this package, let's be consistent, and use `metav1`

File: cmd/svcat/output/output.go [hunk 3]
```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/output/output.go]: Same as above

File: cmd/svcat/binding/describe_cmd_test.go [hunk 4]
```diff
@@ -21,16 +21,16 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/kubernetes-sigs/service-catalog/cmd/svcat/command"
-	"github.com/kubernetes-sigs/service-catalog/cmd/svcat/test"
-	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
-	svcatfake "github.com/kubernetes-sigs/service-catalog/pkg/client/clientset_generated/clientset/fake"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/binding/describe_cmd_test.go]: Same as above

File: cmd/svcat/binding/get_cmd_test.go [hunk 5]
```diff
@@ -21,17 +21,17 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/kubernetes-sigs/service-catalog/cmd/svcat/command"
-	"github.com/kubernetes-sigs/service-catalog/cmd/svcat/test"
-	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
-	svcatfake "github.com/kubernetes-sigs/service-catalog/pkg/client/clientset_generated/clientset/fake"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/binding/get_cmd_test.go]: Same as above

File: cmd/svcat/broker/describe_cmd_test.go [hunk 6]
```diff
@@ -22,15 +22,15 @@ import (
...
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/kubernetes-sigs/service-catalog/pkg/svcat"
 	servicecatalog "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
+	servicecatalogfakes "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/broker/describe_cmd_test.go]: Same as above

File: cmd/svcat/broker/get_cmd_test.go [hunk 7]
```diff
@@ -22,14 +22,14 @@ import (
...
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/kubernetes-sigs/service-catalog/pkg/svcat"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
-	"github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
+	servicecatalog "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
+	servicecatalogfakes "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog/service-catalogfakes"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (jhvhs) [inline cmd/svcat/broker/get_cmd_test.go]: Same as above

File: cmd/svcat/output/binding.go [hunk 8]
```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	svcatsdk "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (jhvhs) [inline cmd/svcat/output/binding.go]: Similar to `metav1`, let's consistently use `corev1` as an alias for consistency and clarity

File: pkg/svcat/svcat.go [hunk 9]
```diff
@@ -30,7 +30,7 @@ type App struct {
 	CurrentNamespace string
 }
 
-// NewApp creates an svcat application.
```
Comment (jhvhs) [inline pkg/svcat/svcat.go]: Same as above

**Review by jhvhs — CHANGES_REQUESTED** [review]
Thank you so much @hezhizhen for this pull request. It tidies up the
code base very nicely.

Could you please address the issues I have outlined in the comments?

File: cmd/svcat/command/command.go (hunk 1 above)
Comment (hezhizhen) [reply cmd/svcat/command/command.go]: I'm still not quite sure if it should be a `an` or `a`, but I'll keep it as it was (`an`).

File: cmd/svcat/instance/provision_cmd_test.go (hunk 2 above)
Comment (hezhizhen) [reply cmd/svcat/instance/provision_cmd_test.go]: I checked every file and converted such v1 into metav1. (I'll skip all the same comments below)

**Pull request closed without merging**